src/Toggle/Object/GetAllToggleStatusRequest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Clearbooks\Labs\Toggle\Object;
+
+use Clearbooks\Labs\Client\Toggle\Entity\Group;
+use Clearbooks\Labs\Client\Toggle\Entity\Segment;
+use Clearbooks\Labs\Client\Toggle\Entity\User;
+
+class GetAllToggleStatusRequest
+{
+    /**
+     * @var User
+     */
+    private $user;
+
+    /**
+     * @var Group
+     */
+    private $group;
+
+    /**
+     * @var Segment[]
+     */
+    private $segments;
+
+    /**
+     * @param User $user
+     * @param Group $group
+     * @param Segment[] $segments
+     */
+    public function __construct( User $user, Group $group, array $segments )
```

Comment:
Ah no, you passin the user segments I guess and then it checks if toggle is enabled for those segments?